### PR TITLE
DOC-Viselements

### LINKED ISSUE #203: Doc: Update viselements.json

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Resolves #203.

### CODE REVIEW

**Review by FabienLelaquais — DISMISSED** [review]
👌
Still some minor things to do... I'm on it

File: src/taipy/gui_core/viselements.json [hunk 1]
```diff
@@ -1,254 +1,256 @@
...
+            "name": "show_add_button",
+            "type": "bool",
+            "default_value": "True",
+            "doc": "If False, the button to create a new scenario is not displayed."
+          },
+          {
+            "name": "display_cycles",
+            "type": "bool",
+            "default_value": "True",
+            "doc": "If False, the cycles are now shown."
```
Comment (FredLL-Avaiga) [inline src/taipy/gui_core/viselements.json]: now => not

File: src/taipy/gui_core/viselements.json [hunk 2]
```diff
@@ -1,254 +1,256 @@
...
+            "doc": "If False, the primary scenarios are not identified with specific visual hint."
+          },
+          {
+            "name": "on_change",
+            "type": "Callback",
+            "doc": "The name of a function that is triggered when the value is updated.<br/>The parameters of that function are all optional:\n<ul>\n<li>state (<code>State^</code>): the state instance.</li>\n<li>var_name (str): the variable name.</li>\n<li>value (<code>Scenario^</code>): the selected scenario.</li>\n</ul>",
+            "signature": [
+              [
+                "state",
+                "State^"
```
Comment (FredLL-Avaiga) [inline src/taipy/gui_core/viselements.json]: waow, does studio builder handle that ?

File: src/taipy/gui_core/viselements.json [hunk 3]
```diff
@@ -1,254 +1,256 @@
...
+            "name": "active",
+            "type": "dynamic(bool)",
+            "default_value": "True",
+            "doc": "Indicates if this component is active.<br/>An inactive component allows no user interaction."
+          },
+          {
+            "name": "expandable",
+            "type": "bool",
+            "default_value": "True",
+            "doc": "If True, the scenario visualizer can be expanded.<br/>If False, the scenario visualizer is collapsed and only its name a <i>submit</i> button are visible."
```
Comment (FredLL-Avaiga) [inline src/taipy/gui_core/viselements.json]: I False, ... is always collapse ?
its name a submit => its name and a submit

File: src/taipy/gui_core/viselements.json [hunk 4]
```diff
@@ -1,254 +1,256 @@
...
+          },
+          {
+            "name": "expanded",
+            "type": "dynamic(bool)",
+            "default_value": "True",
+            "doc": "If True, the scenario visualizer is expanded, and its content is displayed.<br/>If False, the scenario visualizer is collapsed and only its name a <i>submit</i> button are visible."
```
Comment (FredLL-Avaiga) [inline src/taipy/gui_core/viselements.json:91]: name and a submit

**Review by FredLL-Avaiga — COMMENTED** [review]
Well done and a few typos

File: src/taipy/gui_core/viselements.json (hunk 1 above)
Comment (FabienLelaquais) [reply src/taipy/gui_core/viselements.json]: Indeed!

File: src/taipy/gui_core/viselements.json (hunk 2 above)
Comment (FabienLelaquais) [reply src/taipy/gui_core/viselements.json]: Not it does certainly not.

File: src/taipy/gui_core/viselements.json (hunk 3 above)
Comment (FabienLelaquais) [reply src/taipy/gui_core/viselements.json]: On it!

File: src/taipy/gui_core/viselements.json (hunk 4 above)
Comment (FabienLelaquais) [reply src/taipy/gui_core/viselements.json:91]: Yes

**Review by FabienLelaquais — APPROVED** [review]
Finally

**Pull request merged**